File: NewProject/wwwroot/js/Class/Spblfy.js
//商品不良反应
export default class Spblfy {
    constructor() {
        //不良反应ID
        this.blfybh = "";
        //不良反应名称
        this.blfymc = "";
    }
    addData(obj) {
        this.blfybh = obj.blfybh;
        this.blfymc = obj.blfymc;
    }
}

File: NewProject/Models/SPBLFY.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace NewProject.Models
{
    //不良反应
    public class SPBLFY : Table
    {
        [Key]
        public int GUID { get; set; }
        [Display(Name = "不良反应ID")]
        public string BLFYBH { get; set; }
        [Display(Name = "产品ID")]
        public int SPBH { get; set; }
        [Display(Name = "不良反应名称")]
        public string BLFYMC { get; set; }
    }
}
